packages/react-ui-components/src/index.js

```diff
@@ -68,5 +47,9 @@ export default {
     TextInput,
     ToggablePanel,
     Tooltip,
-    Tree
+    Tree,
+    MultiSelectBox,
+    MultiSelectBox_ListPreviewSortable,
```

Comment:
I understand about `MultiSelectBox`, but why would you want to expose smaller internals? These things are sure to be trashed during the next FrankenBox refactoring...